class_rental.rb

```diff
@@ -0,0 +1,20 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  @all_rentals = []
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+    self.class.add_book(self)
+  end
```

Comment:
Good work so far, I would suggest adding the rental to book.rentals and person.rentals to ensure the relation it set in both sides and to be ables to access rentals from book and person instance 

`
 @book = book
    # *The rental is adding itself to the @book achieving the 'belongs-to' association

    book.rentals << self
  
  @person = person

    # *The rental is adding itself to the @person the 'belongs-to' association
    person.rentals << self
`